core/scheduler/service/replay_worker.go

```diff
@@ -97,6 +106,11 @@ func (w *ReplayWorker) startExecutionLoop(ctx context.Context, replayID uuid.UUI
 		}
 		replayWithRun := storedReplayWithRun
 
+		if storedReplayWithRun.Replay.IsTerminated() {
+			w.logger.Info("replay [%s] is externally terminated with status [%s]", replayWithRun.Replay.ID().String(), storedReplayWithRun.Replay.State().String())
```

Comment:
i think the logging format is not consistent with https://github.com/goto/optimus/pull/194/files#diff-c7f1cdd7546059d512af583453b54e1ed17f9498daed33ec1b55f96bac96d5dbR74

Reply:
updated